Chapter_2/2_10_DisturbedClosedOrbits.tex

```diff
@@ -130,47 +130,24 @@ \section{Disturbed closed orbits}\label{sec:2.10}
- \end{align*}
-
- Notice that the previous equation can be rewritten in form $\Delta x_i = \sum_{j=1}^{n_x + n_y}M_{ij}\Delta\theta_j$, which is nothing but a vectorial equation
+ The orbit distortion in the presence of field errors $\delta G(s_j)$ at different positions $s_j$ along the ring is given by:
+	\begin{align*}
+		u_c(s_i) = \dfrac{\sqrt{\beta_u(s_i)}}{2\sin\pi\nu} \sum_j \delta G({s}_j) \sqrt{\beta_u({s}_j)}\cos\{ |\varphi(s_i) - \varphi({s}_j)| - \pi\nu \},
+	\end{align*}
+	where $u$ may represent the $x$ or $y$ coordinate.  The orbit distortions $\Delta u(s_i)$ can be measured in localized positions $s_i$, where the Beam Position Monitors (BPMs) are installed. The equation above can be cast in the form $\Delta u_i = \sum_{j} M_{ij}^{uu} \Delta \theta_{j}^{u}$. Therefore, one can define the orbit response matrix, whose elements can be recognized as:
+ \begin{align}
+ M_{ij}^{uu} = \dfrac{\sqrt{\beta_{u}(s_i)\beta_{u}(s_j)}}{2\sin\left(\pi\nu_{u}\right)}\cos\left( |\varphi_{u}(s_i) - \varphi_{u}(s_j)| - \pi\nu_{u} \right).
+ \end{align}
 
+ The orbit correction system in storage rings has BPMs and small dipoles magnets, called correctors or steering magnets, that steers the beam depending on orbit distortions measured by the BPMs. The orbit distortions measurements and the dipolar kick variations can be sorted in vectors as
  \begin{align*}
-   \Delta \mathbf{x}_{2m} = \mathcal{M}\Delta \boldsymbol{\theta}_{n_x + n_y},
+	 \Delta \vec{u} &= \left(\Delta x_1, \ldots, \Delta x_{\mathrm{N}_{\mathrm{BPM}}}, \Delta y_1, \ldots, \Delta y_{\mathrm{N}_{\mathrm{BPM}}}\right), \\
+	 \Delta \vec{\theta} &= \left(\Delta \theta_1^x, \ldots, \Delta \theta_{\mathrm{N}_{\mathrm{CH}}}^x, \Delta \theta_1^y, \ldots, \Delta \theta_{\mathrm{N}_{\mathrm{CV}}}^y\right).
  \end{align*}
 
- where $\mathcal{M}$ is a $2m \times n_x + n_y$ called orbit response matrix, whose terms are
-
- \begin{align*}
-M_{ij} = \dfrac{\sqrt{\beta_i}}{2\sin\pi\nu}\sqrt{\beta_j}\cos\{ |\varphi_i - \varphi_j| - \pi\nu \}.
-\end{align*}
-
-The distorted closed orbit can be corrected at least where the BPMs are placed and we wish that the displacement produced by the correction magnets cancels out the orbit displacement produced by field errors, i.e., $\mathbf{x}_{2m} + \Delta \mathbf{x}_{2m} = 0$, then $\mathbf{x}_{2m} + \mathcal{M}\Delta\boldsymbol{\theta}_{n_x + n_y} = 0$. Therefore
+ $\mathrm{N}_{\mathrm{BPM}}$ is the number of BPMs, $\mathrm{N}_{\mathrm{CH}}$ is the number of horizontal correctors (CH) and $\mathrm{N}_{\mathrm{CV}}$ is the number of vertical correctors (CV). In this vectorial form, the orbit distortion is related to the kick variation by $\Delta \vec{u} = \mathbf{M} \Delta \vec{\theta}$, and the orbit response matrix has dimension $2 \mathrm{N}_{\mathrm{BPM}} \times \left(\mathrm{N}_{\mathrm{CH}} + \mathrm{N}_{\mathrm{CV}}\right)$.
```

Comment:
... the orbit distortion created by the kicks variation is given by (inline equation), ....